src/Features/CSharp/Portable/Completion/CompletionProviders/AttributeNamedParameterCompletionProvider.cs

```diff
@@ -142,17 +141,16 @@ private bool IsAfterNameEqualsArgument(SyntaxToken token)
             var unspecifiedNamedParameters = attributeNamedParameters.Where(p => !existingNamedParameters.Contains(p.Name));
 
             var text = await semanticModel.SyntaxTree.GetTextAsync(cancellationToken).ConfigureAwait(false);
-            return
-            from p in attributeNamedParameters
-            where !existingNamedParameters.Contains(p.Name)
-            select new CompletionItem(
-                this,
-                p.Name.ToIdentifierToken().ToString() + SpaceEqualsString,
-                CompletionUtilities.GetTextChangeSpan(text, position),
-                CommonCompletionUtilities.CreateDescriptionFactory(workspace, semanticModel, token.SpanStart, p),
-                p.GetGlyph(),
-                sortText: p.Name,
-                rules: ItemRules.Instance);
+            return from p in attributeNamedParameters
+                   where !existingNamedParameters.Contains(p.Name)
+                   select new CompletionItem(
+                       this,
+                       p.Name.ToIdentifierToken().ToString() + SpaceEqualsString,
+                       CompletionUtilities.GetTextChangeSpan(text, position),
+                       CommonCompletionUtilities.CreateDescriptionFactory(workspace, semanticModel, token.SpanStart, p),
+                       p.GetGlyph(),
+                       sortText: p.Name,
+                       rules: ItemRules.Instance);
```

Comment:
Yes, just a formatting change. I struggled to read it initially since it was at the same column as the "return" keyword.